### PR TITLE
Committing the project

### PR DESCRIPTION
**Pull request by MukeshPavan**

### CODE REVIEW

File: FirstOne/src/org/accenture/StreamDemo.java [hunk 1]
```diff
@@ -45,7 +45,7 @@ public static void main(String[] args) {
         System.out.println(minValue);
         Integer maxValue=descSortedMarks.stream().max(Comparator.naturalOrder()).get();
         System.out.println(maxValue);
-        System.out.println("---------------$$$$$$$$$$$$$4445365ew6wtyety-------");
+        System.out.println("------------------");
```
Comment (gayathribheeshma) [inline FirstOne/src/org/accenture/StreamDemo.java]: remove the extra sysouts statements

File: FirstOne/src/org/accenture/StreamDemo.java [hunk 2]
```diff
@@ -45,7 +45,7 @@ public static void main(String[] args) {
         System.out.println(minValue);
         Integer maxValue=descSortedMarks.stream().max(Comparator.naturalOrder()).get();
         System.out.println(maxValue);
-        System.out.println("---------------$$$$$$$$$$$$$4445365ew6wtyety-------");
+        System.out.println("------------------");
         //________________________________________
         //****************************************
```
Comment (gayathribheeshma) [inline FirstOne/src/org/accenture/StreamDemo.java]: Remove this line

File: FirstOne/src/org/accenture/StreamDemo.java [hunk 3]
```diff
@@ -56,23 +48,19 @@ public static void main(String[] args) {
...
             else
                 return s1.compareTo(s2);
-        };
-        List<String> sortedAnimalsBasedOnLength = y.stream().sorted(c).collect(Collectors.toList());
-        System.out.println(sortedAnimalsBasedOnLength);
+        };//method to sort based on length
+        List<String> sortedAnimalsBasedOnLength = y.stream().sorted(c).collect(Collectors.toList());//collecting the sorted animals based on length
+        System.out.println(sortedAnimalsBasedOnLength);//printing the sorted animals
     }
 }
```
Comment (gayathribheeshma) [inline FirstOne/src/org/accenture/StreamDemo.java]: Add new line to avoid this red mark

**Comment by MukeshPavan** [comment]
Updated.

**Pull request merged**